EKF/common.h

```diff
@@ -225,6 +225,11 @@ struct parameters {
 	float tas_innov_gate;		// True Airspeed Innovation consistency gate size in standard deciation
   	float eas_noise;			// EAS measurement noise standard deviation used for airspeed fusion [m/s]
 
+  	// syntetic sideslip fusion
+  	float beta_innov_gate;		// Syntetic sideslip innovation consistency gate size in standard deviation
```

Comment:
'Synthetic' spelling. Also please use same convention as other parameters and append units in parenthesis, eg (STD)